src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -10,15 +10,29 @@ public class WorkWithFile {
     public static final String NAME_OF_SUPPLY = "supply";
     public static final int OPERATION_INDEX = 0;
     public static final int AMOUNT_INDEX = 1;
+    public static final String REGEX_SPLIT = "[,\\s]";
+    public static final String STATISTIC_PATTERN = "supply,%d%n"
+            + "buy,%d%n"
+            + "result,%d%n";
+    public static final int INITIAL_SUPPLY_AMOUNT = 0;
+    public static final int INITIAL_BUY_AMOUNT = 0;
```

Comment:
Useless constant. When you create a variable to count smth, just define it as 0.